core/modelspec/creates.js

```diff
@@ -17,7 +17,7 @@ import {ModelPublishMode, ModelSpec} from './ModelSpec';
  * for access by all sub-components.
  *
  * The model created will be considered to be 'owned' by the receiving component. If it implements
- * `@LoadSupport` it will be loaded on component mount, and it will always be destroyed on
+ * `LoadSupport` it will be loaded on component mount, and it will always be destroyed on
```

Comment:
Here's another reference to `LoadSupport` - I'm still leaning towards `hasLoadSupport` as the getter name. (Same in `uses()` and `RefreshContextModel` docs.)